addons/base_iban/static/src/components/iban_widget/iban_widget.js

```diff
@@ -2,26 +2,29 @@ import { registry } from "@web/core/registry";
 import { CharField, charField } from "@web/views/fields/char/char_field";
 import { useDebounced } from "@web/core/utils/timing";
 import { useService } from "@web/core/utils/hooks";
-import { useState } from "@odoo/owl";
+import { useState, onMounted, onPatched } from "@odoo/owl";
 
 export const DELAY = 400;
 
 export class IbanWidget extends CharField {
     static template = "base_iban.iban";
     setup() {
         super.setup();
-        this.state = useState({ isValidIBAN: null });
+        this.state = useState({ ibanLabel: "" });
         this.orm = useService("orm");
         this.validateIbanDebounced = useDebounced(async (ev) => {
             const iban = ev.target.value;
-            if (!iban) {
-                this.state.isValidIBAN = null;
-            } else if (!/[A-Za-z]{2}.{3,}/.test(iban)) {
-                this.state.isValidIBAN = false;
-            } else {
-                this.state.isValidIBAN = await this.orm.call("res.partner.bank", "check_iban", [[], iban]);
-            }
+            await this.validateIban(iban);
         }, DELAY);
+
+        onMounted(this.validateIban);
+        onPatched(this.validateIban);
+    }
+
+    async validateIban(
```

Comment:
in JS it's validateIban, but in python it could be validating IBAN or CLABE?

Reply:
I tend to agree, but base_iban seems to apply constraints on the account number - we might not want that